### PR TITLE
Change customer request flow

### PR DESCRIPTION
**Pull request by Namiii**
Implemented the new Request builder flow for Customer request

### CODE REVIEW

**Comment by Namiii** [comment]
@pitiphong-p I'm not so sure what you mean, the customer model has a `default_card` variable, is there something else that needs to be added?

**Comment by pitiphong-p** [comment]
@Namiii Oh, sorry, false alarm

**Comment by Namiii** [comment]
@pitiphong-p hahaha alright :D

File: src/test/java/co/omise/live/LiveCustomerRequestTest.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+import co.omise.models.Ordering;
+import co.omise.models.ScopedList;
+import co.omise.requests.Request;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static junit.framework.TestCase.assertEquals;
```
Comment (nuxzero) [inline src/test/java/co/omise/live/LiveCustomerRequestTest.java]: I think we should use `org.junit` in Junit 4 stead of using `junit.framework` in Junit 3.

File: src/test/java/co/omise/requests/CustomerRequestTest.java [hunk 2]
```diff
@@ -51,8 +42,11 @@ public void testCreate() throws IOException, OmiseException {
...
-        Customer customer = resource().update(CUSTOMER_ID, new Customer.Update()
-                .email("john.doe.the.second@example.com"));
+        Request<Customer> request = new Customer.UpdateRequestBuilder(CUSTOMER_ID)
+                .email("john.doe.the.second@example.com")
+                .build();
+        Customer customer = getTestRequester().sendRequest(request);
+
         assertRequested("PATCH", "/customers/" + CUSTOMER_ID, 200);
 
         assertEquals("cust_test_4yq6txdpfadhbaqnwp3", customer.getId());
```
Comment (nuxzero) [inline src/test/java/co/omise/requests/CustomerRequestTest.java]: Is this `cust_test_4yq6txdpfadhbaqnwp3` same as `CUSTOMER_ID`?

File: src/test/java/co/omise/live/LiveCustomerRequestTest.java (hunk 1 above)
Comment (Namiii) [reply src/test/java/co/omise/live/LiveCustomerRequestTest.java]: Yeah right, I need to be more careful with my imports 🤦‍♂️

File: src/test/java/co/omise/requests/CustomerRequestTest.java (hunk 2 above)
Comment (Namiii) [reply src/test/java/co/omise/requests/CustomerRequestTest.java]: Yes should have used `CUSTOMER_ID`, will make changes

**Pull request merged**